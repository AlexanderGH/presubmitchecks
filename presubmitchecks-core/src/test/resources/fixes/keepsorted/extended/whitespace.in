Initial extra indent is not ignored:
    // keep-sorted-test start
        b
      c
      a
    // keep-sorted-test end
    middle
    // keep-sorted-test start
        b
    c
    a
    // keep-sorted-test end