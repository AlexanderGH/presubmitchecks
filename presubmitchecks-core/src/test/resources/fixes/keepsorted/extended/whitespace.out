Initial extra indent is not ignored:
    // keep-sorted-test start
        b
      a
      c
    // keep-sorted-test end
    middle
    // keep-sorted-test start
        b
    a
    c
    // keep-sorted-test end
